### PR TITLE
[WIP] Adding HT binned WJetsToQQ and ZJetsToQQ cards at LO_MLM and NLO for EXO backgrounds request

### PR DESCRIPTION
**Pull request by cmantill**
- LO cards are based in legacy production:
https://github.com/cms-sw/genproductions/blob/pre2017/bin/MadGraph5_aMCatNLO/cards/production/13TeV/ZJets_LO_MLM/ZJetsToQQ_HT-600toInf/ZJetsToQQ_HT600toInf_proc_card.dat
https://github.com/cms-sw/genproductions/blob/pre2017/bin/MadGraph5_aMCatNLO/cards/production/13TeV/WJets_HT_LO_MLM/WJetsToQQ_HT-600toInf/WJetsToQQ_HT-600toInf_proc_card.dat

- Only 600toInf HT bin existent in legacy production, replaced by 3 HT bins: 400-600, 600-800,800toInf. 

- NLO cards (non-existent in legacy production) are based on LO cards but added madspin card to define e.g. z > j j syntax.

- Cards tested by parsing code but received following warnings:
WARNING in Proc/Run Card: plese use the correct pdf with b quark (nf==5)
WARNING in Proc Card: in 'add process line' NO same definition of the process that is in 'generate' line. 
WARNING in Proc Card: no/wrong output line in the proc card. Plese add this line at the END of card: 'output name_of_process -nojpeg'
but the cards seem fine, is it a false alarm or are any modifications needed?

- LO legacy proc cards discarded higgs diagrams in "$$ h", is this relevant? should we keep it or not?

### CODE REVIEW

**Comment by perrozzi** [comment]
The HT binning at NLO is not well defined since an additional LO jet is added as real radiation from the NLO correction.
I believe the samples would be impossible to use together with the inclusive ones.
@kdlong @qliphy can you confirm?

**Comment by cmantill** [comment]
Which inclusive ones? is there any plan to produce inclusive ones (i.e no HT binning) at NLO? We were also thinking on producing only an inclusive sample at NLO but weighted with a similar bias weight function as in here:

https://github.com/cms-sw/genproductions/blob/mg26x/bin/MadGraph5_aMCatNLO/cards/examples/dyellell012j_5f_NLO_FXFX_bias/dyellell012j_5f_NLO_FXFX_bias_cuts.f#L901-L976

would that work then? What about LO? does those seem fine?

**Comment by perrozzi** [comment]
HT binning at LO is well defined and the samples to be produced with these datacards can be consistenly merged with the inclusive LO sample.
For the NLO case this is not possible, which means a very limited usage of the samples (yes, we always produce a bit of inclusive).
The bias weight will become the standard way to produce these requests.
It is under testing, you are gladly invited to contribute!

**Comment by perrozzi** [comment]
please remove the NLO datacards for this PR to be merged

**Comment by cmantill** [comment]
NLO cards have been removed. I am currently producing LO gridpacks in cms connect and testing NLO cards with bias weight using mg6x branch. Are there any approved examples of this? The suggestion would be still to do HT bins in this way?

**Comment by perrozzi** [comment]
please follow these instructions to cleanup the datacards
https://twiki.cern.ch/twiki/bin/view/CMS/Moriond18MC#Checks_to_the_datacards_BEFORE_T

concerning the bias weight, an example is here
https://github.com/cms-sw/genproductions/tree/mg26x/bin/MadGraph5_aMCatNLO/cards/examples/dyellell012j_5f_NLO_FXFX_bias

**Comment by cmantill** [comment]
I hope now the datacards are cleaned

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/WJets_LO_MLM/WJetsToQQ_HT-800toInf/WJetsToQQ_HT-800toInf_proc_card.dat [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import model sm-ckm_no_b_mass
+#switch to diagonal ckm matrix if needed for speed
+#import model sm-ckm_no_b_mass
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/WJets_LO_MLM/WJetsToQQ_HT-800toInf/WJetsToQQ_HT-800toInf_proc_card.dat]: this needs to be removed

**Comment by cmantill** [comment]
Sorry about that, its done now.

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ZJets_LO_MLM/ZJetsToQQ_HT-800toInf/ZJetsToQQ_HT800toInf_proc_card.dat [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import model sm-ckm_no_b_mass
+define q = j 
+
+define ell+ = e+ mu+ ta+
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ZJets_LO_MLM/ZJetsToQQ_HT-800toInf/ZJetsToQQ_HT800toInf_proc_card.dat]: this is not used and can be removed

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ZJets_LO_MLM/ZJetsToQQ_HT-800toInf/ZJetsToQQ_HT800toInf_proc_card.dat [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import model sm-ckm_no_b_mass
+define q = j
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ZJets_LO_MLM/ZJetsToQQ_HT-800toInf/ZJetsToQQ_HT800toInf_proc_card.dat]: this is not needed. also for the w case you can use j and remove q...

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ZJets_LO_MLM/ZJetsToQQ_HT600to800/.log [hunk 4]
```diff
@@ -0,0 +1,133 @@
+Starting job on  Thu Sep 28 23:05:09 CEST 2017
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ZJets_LO_MLM/ZJetsToQQ_HT600to800/.log]: is there a reason to keep this log file?

**Comment by cmantill** [comment]
removed not used definitions and log file

**Pull request merged**